### PR TITLE
design change

### PR DESCRIPTION
**Pull request by nazaninreihani**

### CODE REVIEW

File: src/javascript/common/header.js [hunk 1]
```diff
@@ -13,9 +13,11 @@ const Header = (function () {
     const userMenu = function() {
         if (!Client.is_logged_in()) {
             $('#main-login').removeClass('hidden');
+            $('#main-signup').removeClass('hidden');
```
Comment (mohammad-deriv) [inline src/javascript/common/header.js]: What do you think about `$('#main-login, #main-signup').removeClass('hidden');`?

File: src/templates/layout/includes/slider.html.tt [hunk 2]
```diff
@@ -0,0 +1,54 @@
+<div class="slider">
+    <div id="slide-1">
+      <div class="fx-section center-text slider-text">
+          <div class="container">
+              <div class="row">
+                  <div class="col-sm col-md-10 offset-md-1 col-lg-8 offset-lg-2 flex-md-middle flex-sm-middle flex-lg-middle flex-xs-middle">
+                      <h3 class="bold white">[% l('ChampionFX: Where the Client is King') %]</h3>
```
Comment (mohammad-deriv) [inline src/templates/layout/includes/slider.html.tt]: Can we use `<strong>` here and remove the `.bold` declaration?

**Review by mohammad-deriv — CHANGES_REQUESTED** [review]

File: src/templates/layout/includes/slider.html.tt (hunk 2 above)
Comment (nazaninreihani) [reply src/templates/layout/includes/slider.html.tt]: yes sure

File: src/sass/champion.scss [hunk 3]
```diff
@@ -40,39 +54,70 @@ html {
...
+}
+
 @media (max-width: 480px) {
     .signup-box input {
         margin: 15px 0;
     }
     #end_note {
         display: none;
     }
+    #license-box-container {
```
Comment (mohammad-deriv) [inline src/sass/champion.scss]:
```css
#end_note, #license-box-container {
    display: none;
}
```

File: src/sass/champion_overwrite.scss [hunk 4]
```diff
@@ -1,7 +1,34 @@
...
+.logo > div {
+    height: 45px;
+}
+
+.binary-logo-text > div {
+    height: 45px;
+}
+
+.logo-wrapper .logo-parent .logo, .logo-wrapper .logo-parent .logo-dark {
+    height: 45px;
```
Comment (mohammad-deriv) [inline src/sass/champion_overwrite.scss]: Can we have all the above selectors having `height: 45px;` together?

File: src/sass/slider.scss [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+        width: 100%;
+    }
+}
+
+.slider-text {
+    padding: 30px;
+    .container {
+        height: 100%;
+        .row {
+            height: 100%;
```
Comment (mohammad-deriv) [inline src/sass/slider.scss]:
```css
.container {
    &, .row {
        height: 100%;
    }
}
```

File: src/templates/layout/includes/header.html.tt [hunk 6]
```diff
@@ -63,10 +54,125 @@
...
+                    <ul class="signup-button">
+                        <li class=""><button class="button-secondary" href="javascript:;">[% l('Sign Up') %]</button></li>
+                    </ul>
                 </div>
             </div>
+            <div class="col-lg-7 pull-lg-2">
+                <ul class="top-nav-menu">
+                    <li>
+                      <a href="javascript:;" class="nav-dropdown-toggle">
+                      [% l('About') %]
```
Comment (mohammad-deriv) [inline src/templates/layout/includes/header.html.tt]: Please check the indentations of the changes in this PR as well.
Thanks.

**Review by mohammad-deriv — CHANGES_REQUESTED** [review]

File: src/sass/champion_overwrite.scss (hunk 4 above)
Comment (nazaninreihani) [reply src/sass/champion_overwrite.scss]: isn't it just the logo?

**Review by mohammad-deriv — APPROVED** [review]

**Pull request merged**